content/en/storage-providers/operate/index-provider.md

```diff
@@ -65,9 +65,16 @@ The exact storage usage depends on the number of multihashes stored in a single
 
 Based on the equation above, caching 128-bit long multihashes will result in chunk sizes of 0.25MiB with maximum cache growth of 256 MiB for advertisements with a single chunk.
 
-## First-time migration
+## Become a Index Provider
 
-A one-off migration is needed to enable index provider migration to re-generate DagStore CARv2 indices. To enable the indexing integration, follow the following step:
+{{< alert icon="tip" >}}
+This feature is available in lotus v1.15.1 and up.
+{{< /alert >}}
+
+If you are a new Storage Provider created on or after v1.15.1, you can move directly to step 11 in the [one-off migration](#one-off-migration) section after you have finished your first storage deal.
```

Comment:
"running on v1.15.1 or greater"

Also please make sure the link works.

Reply:
I should maybe rethink the formulation here. The idea was to clarify that Storage Providers created after v1.15.1 will have the new CARv2 indexing format from the beginning, and therefore does not need to do the one-off migration. "Running on v1.15.1 or greater" could also mean established Storage Providers with the old dagstore.

Reply:
I see. Maybe we could have a branching of steps right at the top and
separate steps for existing and new storage providers?

Reply:
Yeah, I will modify and create separate steps for existing and new storage in a new PR. I closed this one due to a missing ds store file on my side which I did not manage to commit back properly.